.ci/jobs/defaults.yml

```diff
@@ -47,21 +47,3 @@
     publishers:
       - email:
           recipients: infra-root+build@elastic.co
-      - junit:
```

Comment:
For my understanding, is this file remaining due to https://github.com/elastic/rally/blob/master/.ci/jobs/release-docker.yml job?

Reply:
Yes, exactly. I'm not sure what exactly was needed or not so I left most of it.